src/main/java/com/training/tinyurl/service/ITinyUrlService.java

```diff
@@ -2,7 +2,10 @@
 
 import com.training.tinyurl.dto.RegistrationReqDto;
 import com.training.tinyurl.exceptionhandler.MongoApiException;
+import com.training.tinyurl.security.AppUserDetails;
 
 public interface ITinyUrlService {
     public void createNewUser(RegistrationReqDto request) throws MongoApiException;
+    public AppUserDetails getLoggedInUserDetails();
```

Comment:
This is implementation specific method. Should not be in interface.
It should be private method and either in ServiceImpl or ControllerImpl as per usage.
Or Service should contain loginUser and logoutUser methods which should internally call this method.